Update GitHub issue templates (#1120)
huggingface/hub-docs: .github/ISSUE_TEMPLATE/bugs.md

---
name: Bug
about: Report a bug you face in the Hub
title: ''
labels: ''
assignees: ''

---

> **This repository is focused on the Hub experience and documentation. If you're facing an issue with a specific library, please open an issue in the corresponding repo**

Reviewer: ```suggestion
**This repository is focused on the Hub experience and documentation. If you're facing an issue with a specific library, please open an issue in the corresponding GitHub repo. If you're facing an issue with a specific model or dataset, please open an issue in the corresponding HF repo.**
```

Reviewer: (maybe)

**Bug description.**
A clear and concise description of what the problem is. Ex. Clicking this button is not working when [...]

**Describe the expected behaviour**
A clear and concise description of what you want to happen.

**Additional context**

> Add any other context or screenshots about the feature request here. Please share details such as browser when it's relevant.

Reviewer 2: ```suggestion
Add any other relevant context or screenshots here. Please share details such as browser when appropriate.
```
